src/require.jl

```diff
@@ -85,6 +85,15 @@ function __init__()
           NLopt.upper_bounds!(opt, upper_bounds)
           NLopt.maxeval!(opt, maxeval)
 
+          if nstart > 1
+            localopt = opt
+            opt = NLopt.Opt(:G_MLSL_LDS, length(lower_bounds))
+            NLopt.lower_bounds!(opt, lower_bounds)
+            NLopt.upper_bounds!(opt, upper_bounds)
+            NLopt.local_optimizer!(opt, localopt)
+            NLopt.maxeval!(opt, nstart * maxeval)
```

Comment:
Only these would need to be inside the conditional and the repeated code can be avoided